doc/whats_new.rst

```diff
@@ -363,8 +366,8 @@ Changes
 
 - All the connectivity examples are changed from ADHD to brain development
   fmri dataset.
-- Examples plot_decoding_tutorial, plot_haxby_decoder, 
```

Comment:
if your editor removes trailing whitespace could you configure it to modify only lines you touched? thanks!

Reply:
I am not aware of such a configuration existing in VSC.